docs/index.rst

```diff
@@ -2,42 +2,64 @@
 sunpy core Documentation
 ************************
 
-The sunpy core package is a community-developed, free and open-source solar
+``sunpy`` is a community-developed, free and open-source solar
 data analysis environment for Python.
 
+New Users
+*********
+
+If you are a first time user or new to Python,
+
+..
+        * :Introduction to sunpy core [new page described in #6615]
+
+* :doc:`installation guide <intro/installation>`
```

Comment:
We should avoid using the `:doc:` role where possible and use `:ref:` with the reference names of the titles we are referencing. This is so that if we change the path the links still work.